docs/community/100-days-of-oss.md

```diff
@@ -3,79 +3,100 @@ id: 100-days-of-oss
-- [Tweet out your commitment today](https://twitter.com/intent/tweet?text=I%27m%20joining%20the%20%23100DaysOfOSS%20challenge%20with%20%40saucedopen%20to%20grow%20my%20skills%2C%20gain%20real-world%20experience%20in%20open%20source%2C%20and%20to%20make%20a%20positive%20impact%20on%20the%20OSS%20Community!) or share on your platform of choice!
+
+- [Sign up](https://docs.google.com/forms/d/e/1FAIpQLScoq6ORpkzUg3Cqxd3EeNnMP6HIJ3uemPhSHs8sYZALmbpeVg/viewform?usp=sharing) with OpenSauced to receive a coupon code for 12 months of free access to all of OpenSauced's paid features and for the chance to opt in to receive updates, event invites, and resources to help you succeed.
+- [Post your commitment today on X](https://twitter.com/intent/tweet?text=I%27m%20joining%20the%20%23100DaysOfOSS%20challenge%20with%20%40saucedopen%20to%20grow%20my%20skills%2C%20gain%20real-world%20experience%20in%20open%20source%2C%20and%20to%20make%20a%20positive%20impact%20on%20the%20OSS%20Community!) or share on your platform of choice!
 
 ## The Official Kickoff
+
 We're starting 100 days on July 23rd to the end of [Hacktoberfest](https://hacktoberfest.com/), a month-long celebration of open source contributions. We'll provide continuous support, daily inspirational tweets, and engaging events to help you stay motivated and make progress.
 
-Don't worry if you're unable to start on the same day as everyone else. The #100DaysOfOSS challenge is flexible, and you can join in whenever you're ready. Just jump in at any point and begin with day one of your personal challenge. 
+Don't worry if you're unable to start on the same day as everyone else. The #100DaysOfOSS challenge is flexible; you can join in whenever you're ready. Just jump in at any point and begin with day one of your personal challenge.
 
 ## Support
+
 To make the most of your #100DaysOfOSS journey, here are some additional resources and events you can explore:
 
-1. **Weekly Twitter Spaces:** Join our weekly Twitter Spaces sessions where we discuss open source topics, share insights, and connect with like-minded individuals. Follow us on [Twitter](https://twitter.com/saucedopen) to stay updated on upcoming sessions.
+1. **Weekly X Spaces:** Join our weekly X Spaces sessions, where we discuss open source topics, share insights, and connect with like-minded individuals. Follow us on [X](https://twitter.com/saucedopen) to stay updated on upcoming sessions.
 
 2. **Community Events:** Discover a wide range of events on our [community docs page](https://docs.opensauced.pizza/community/welcome-to-the-community). Whether it's hack days, workshops, or office hours, these events provide excellent opportunities to learn, collaborate, and find new projects to contribute to.
```

Comment:
Currently, the link goes to the "Welcome to the Community" page. And we don't have events listed in that page.